code/modules/food_and_drinks/recipes/recipes_microwave.dm

```diff
@@ -476,10 +476,153 @@
+		/obj/item/reagent_containers/food/snacks/breadslice,
+		/obj/item/reagent_containers/food/snacks/cutlet,
+		/obj/item/reagent_containers/food/snacks/cheesewedge,
+		/obj/item/reagent_containers/food/snacks/grown/cabbage
+	)
+	result = /obj/item/reagent_containers/food/snacks/philly_cheesesteak
+
+/datum/recipe/microwave/ppattyred
+	reagents = list("cherryjelly" = 5)
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/reagent_containers/food/snacks/meat
+	)
+	result = /obj/item/reagent_containers/food/snacks/ppattyred
+
+/datum/recipe/microwave/ppattyorange
+	reagents = list("orangejuice" = 5)
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/reagent_containers/food/snacks/meat
+	)
+	result = /obj/item/reagent_containers/food/snacks/ppattyorange
+
+/datum/recipe/microwave/ppattyyellow
+	reagents = list("lemonjuice" = 5)
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/reagent_containers/food/snacks/meat
+	)
+	result = /obj/item/reagent_containers/food/snacks/ppattyyellow
+
+/datum/recipe/microwave/ppattygreen
+	reagents = list("limejuice" = 5)
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/reagent_containers/food/snacks/meat
+	)
+	result = /obj/item/reagent_containers/food/snacks/ppattygreen
+
+/datum/recipe/microwave/ppattyblue
+	reagents = list("berryjuice" = 5)
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/reagent_containers/food/snacks/meat
+	)
+	result = /obj/item/reagent_containers/food/snacks/ppattyblue
+
+/datum/recipe/microwave/ppattypurple
+	reagents = list("grapejuice" = 5)
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/reagent_containers/food/snacks/meat
+	)
+	result = /obj/item/reagent_containers/food/snacks/ppattypurple
+
+/datum/recipe/microwave/ppattywhite
+	reagents = list("sugar" = 5)
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/reagent_containers/food/snacks/meat
+	)
+	result = /obj/item/reagent_containers/food/snacks/ppattywhite
+
+/datum/recipe/microwave/ppattyrainbow
+	items = list(
+		/obj/item/reagent_containers/food/snacks/ppattyred,
+		/obj/item/reagent_containers/food/snacks/ppattyorange,
+		/obj/item/reagent_containers/food/snacks/ppattyyellow,
+		/obj/item/reagent_containers/food/snacks/ppattygreen,
+		/obj/item/reagent_containers/food/snacks/ppattyblue,
+		/obj/item/reagent_containers/food/snacks/ppattypurple,
+		/obj/item/reagent_containers/food/snacks/ppattywhite
+	)
+	result = /obj/item/reagent_containers/food/snacks/ppattyrainbow
+
+/datum/recipe/microwave/elecburger
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/stack/sheet/mineral/plasma,
+		/obj/item/stack/sheet/mineral/plasma
+	)
+	result = /obj/item/reagent_containers/food/snacks/elecburger
+
+/datum/recipe/microwave/ratburger
+	items = list(
+		/obj/item/reagent_containers/food/snacks/bun,
+		/obj/item/holder/mouse
```

Comment:
im going to put living mice in the microwave, this is gonna be great

Reply:
Now more than just diona nymphs can receive the microwave treatment.